src/Bicep.Core.Samples/Files/InvalidLoadFunctions_CRLF/Assets/encoding-iso.txt

```diff
@@ -1,94 +1,94 @@
-�		inverted exclamation mark
```

Comment:
Even with the "Hide whitespace changes" option enabled, Github is showing every line of this file as changed but is unable to identify the alteration. Is the editor you used maybe not handling this text encoding correctly?